scripts/build.sh

```
@@ -0,0 +1,113 @@
+fi
+
+if [ -f "*.vsix" ]; then
+    rm *.vsix
+    info "Removed existing .vsix files"
+fi
+
+# Step 4: Generate themes
+info "Generating theme files from base configuration..."
+npm run build
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)